airbyte-webapp/src/components/connection/CatalogTree/next/CatalogTreeTableRow.tsx

```diff
@@ -63,14 +63,13 @@ export const CatalogTreeTableRow: React.FC<StreamHeaderProps> = ({
     [styles.disabledChange]: changedSelected && !isStreamEnabled,
     [styles.selected]: isSelected,
     [styles.error]: hasError,
+    [styles.disabled]: !changedSelected && !isStreamEnabled,
```

Comment:
Note that this makes the syncmode dropdown blend into the background.  I do see this in the designs as well, and users can't save changes to a disabled stream anyways currently... so maybe that's ok?  Since they can't change things, I wonder if we want to disable the dropdowns in these rows though.

Reply:
I noticed that! I'm not a big fan, and maybe we can work around it by disabling it. Additionally, I feel the color contrast may be too low and we may need to revisit this with design.